doc/changelog.d/3263.fixed.md

```diff
@@ -0,0 +1 @@
+fix: not showing instances on linux
```

Comment:
Why is there 2 changelog fragments for this PR?

Reply:
You can delete "manually" the fragment of the secondary PR if you prefer.

Reply:
Agreed. I'm removing it.

Does it make sense to have that fragment though? Should the bot only generate that fragment when the PR is targetting main? I do also acknowledge that the PR was targetting main, but then I did switch it.

Reply:
That is something you can control on your workflow based on your preferences. I would keep ALL fragments though, since it doesn't do any harm. But I get that sometimes you might not want to create the fragment. You can also implement this logic via labels if you prefer.

Reply:
In the sense: "if label 'no-fragment' is added, skip fragment generation/delete it".

Reply:
Got it. Thank you @RobPasMue !

Reply:
Anytime!